nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -183,14 +183,15 @@ let
                 ${flip concatMapStrings ips (ip:
                   let
                     cidr = "${ip.address}/${toString ip.prefixLength}";
+                    brd = if ip.broadcast == null then "+" else ip.broadcast;
```

Comment:
I think this fails to evaluate for ipv6 because only ipv4 will have the broadcast attribute set to null by default.